### PR TITLE
feat(monitoring): SD-3413 Added keyboard shortcuts for starting item actions

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
SD-3413

Also fixed the commented issue for SD-3411 and SD-3412.

### CODE REVIEW

File: client/app/scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -2201,6 +2203,7 @@
                     cssClass: 'upload-avatar modal-static modal-responsive',
                     controller: ChangeImageController,
                     templateUrl: 'scripts/superdesk-authoring/views/change-image.html',
+                    keyboardShortcut: 'ctrl+e',
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-authoring/authoring.js]: Shouldn't this be a different key? This is already used by 'Edit'.

**Comment by sivakuna-aap** [comment]
Do we really want to override the cut and paste shortcuts? They would be needed by users.

**Comment by plamut** [comment]
Overriding common shortcuts should indeed be avoided IMO, it hinders usability.

File: client/app/scripts/superdesk-monitoring/monitoring.js [hunk 2]
```diff
@@ -336,6 +373,8 @@
                     scope.selected = item;
                     monitoring.selectedGroup = scope.group;
                     monitoring.preview(item);
+
+                    bindActionKeyShortcuts(item);
```
Comment (petrjasek) [inline client/app/scripts/superdesk-monitoring/monitoring.js]: you should also unbind it - `scope.$on('$destroy', unbindActionKeyShortcuts);`

File: client/app/scripts/superdesk-authoring/authoring.js (hunk 1 above)
Comment (pavlovicnemanja) [inline client/app/scripts/superdesk-authoring/authoring.js]: Yeah, my bad, I accidentally added keyboard shortcut for this action also.

**Comment by pavlovicnemanja** [comment]
@plamut @sivakuna-aap @petrjasek What is your suggestion? Maybe to use ctrl + shift + {key} ?

**Comment by pavlovicnemanja** [comment]
I disabled item action shortcuts while user is working with text.

**Comment by ioanpocol** [comment]
:+1:

**Comment by Aca-jov** [comment]
:+1:

**Pull request merged**